docs/guides/runtime-configuration.rst

```diff
@@ -1,61 +1,38 @@
 Runtime Configuration
 =====================
 
-.. decide where to expose this in the docs
+Most productive things you can do with Orquestra Workflow SDK require interacting with runtimes.
+A *runtime* is execution environment for workflows; it knows about your workflow runs, can host web UIs, stores Orquestra secrets.
 
-Orquestra can support different execution environments for workflows, called runtimes.
-Currently there are two supported runtimes: local execution via Ray, and remote execution via Compute Engine.
+Runtimes fall into two general categories:
 
-In some cases, additional configuration options are required in order to use a runtime.
-For example, a URL is required to connect to Compute Engine.
-Choosing a runtime and supplying options is called a *Runtime Configuration*.
+#. *Remote*. Orquestra Compute Engine. Managed clusters.
+#. *Local*. These runtimes support a subset of Orquestra functionality for local use.
 
-There is always one *Runtime Configuration* defined, called ``local``.
-This configuration option is reserved and cannot be updated or saved to.
-Manually editing the configuration file will not change this reserved option.
+Runtime Selection
+-----------------
 
-This is used by default in the CLI and executes a workflow locally with the default runtime options.
+Many of Workflow SDK APIs need specifying the runtime to use. Runtimes are identified by **config name**.
 
-..
-    TODO: Add how CLI uses configurations
+* ``auto`` -- guesses the runtime to use based on contextual information. See the section below for more information.
+* ``in_process`` -- refers to a local runtime that executes tasks sequentially in the same Python process. Useful for debugging.
+* ``ray`` -- refers to a local Ray runtime that executes workflows in the background. Allows running tasks in parallel. Requires starting the runtime daemon with ``orq up``.
+* ``local`` -- alias for ``ray``. Created at a time when there was just a single local runtime kind.
```

Comment:
Suggestion: our users probably don't need to know the history.
```suggestion
* ``local`` -- alias for ``ray``.
```

I think there's a wider discussion to be had about whether we want to sunset `local` (or leave it but start omitting it from docs), but that's beyond the scope here :)